components/tidb_query/src/codec/row/v2.rs

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019 TiKV Project Authors. Licensed under Apache-2.0.
+
+use std::{i16, i32, i8, u16, u32, u8};
+
+pub const CODEC_VERSION: u8 = 128;
```

Comment:
Maybe adding a `MAGIC` would be better, since `v2` conflicts to `version: u8 = 128` semantically. By calling it a magic number it makes sense.

Reply:
Do you mean `MAGIC_CODE_VERSION`?

Reply:
Yes maybe.